### PR TITLE
Feature missing alderman

### PR DESCRIPTION
**Pull request by Coqueiro**
This PR references issue #29.

- [x] Fix alderman nickname (not getting special characters)

- [x] Querying mongo to get list of alderman

- [ ] 100% Coverage

- [x] Resolve merge conflicts

- [x] Store present alderman using their ID (this branch is storing the missing ones by their names, until this point)

### CODE REVIEW

**Comment by villasv** [comment]
Even though this address #29, it technically does the opposite: collect the missing names.
Doing the former makes more sense in the general purpose, and it should be stored as a list of IDs.

The tools module itself seems unnecessary, as its not independent/resusable enough to deserve to be separate.

File: vigilante/spiders/alderman.py [hunk 1]
```diff
@@ -36,9 +36,9 @@ def parse_alderman(response):
         'id',
         int(parse.parse_qs(parse.urlparse(response.url).query)['cvd'][0])
     )
-    loader.add_xpath(
+    loader.add_value(
         'nickname',
-        '//div[@class="titulo"]/p[@class="titnot"]/text()'
+        parse.parse_qs(parse.urlparse(response.url).query)['nome_politico'][0]
```
Comment (villasv) [inline vigilante/spiders/alderman.py]: Why was this necessary? This is not bad, assuming that the URL always uses the stored nickname, but assumptions come with risk.

File: tests/test_minute_spider.py [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 from unittest import TestCase
+from unittest.mock import patch
 from scrapy.http import Request, TextResponse
-from vigilante.spiders.minutes import MinutesSpider, parse_minutes
+from vigilante.spiders.minute import MinuteSpider
 
 
-class MinutesSpiderTests(TestCase):
+class MinuteSpiderTests(TestCase):
+    @patch.dict('vigilante.spiders.minute.os.environ', {'MONGODB_URI' : 'mongomock://localhost' })
```
Comment (villasv) [inline tests/test_minute_spider.py]: Unit tests do not have service dependency. Mocking a database is responsibility of integration tests.

File: vigilante/spiders/minute.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+from vigilante.items.alderman import Alderman
+from vigilante.items.minute import MinuteLoader
+
+
+class MinuteSpider(scrapy.Spider):
+    def __init__(self):
+        mongo_url = os.environ['MONGODB_URI']
+        self.alderman_list = []
+        connect(host=mongo_url)
+        for alderman in Alderman.objects:
```
Comment (villasv) [inline vigilante/spiders/minute.py]: Why not just convert it to list. Are you trying to save some memory here?

File: vigilante/spiders/alderman.py (hunk 1 above)
Comment (Coqueiro) [inline vigilante/spiders/alderman.py]: We are already making assumptions over html structure... but I get your point. I've done this to get the name correctly. I'm checking now and it seems ok, but yesterday the web page was showing names in a wrong format. For example, JoÃ£o instead of João. I thought URL was more reliable.

Comment (villasv) [inline vigilante/spiders/alderman.py]: Assumptions over the structure are inevitable. This one is not. But if the web page is not reliable, of course we should resort to something else. Got any links with examples of bad names?
